app/components/LongCode.tsx

```diff
@@ -50,11 +50,22 @@ const LongCode: FC<LongCodeProps> = ({ code, codeClass }: LongCodeProps) => {
   return (
     <Box data-testid="long-code-code" className={codeClass} aria-hidden="true">
       {codeLines.map((line, i) => {
-        return (
-          <Box component="span" key={[line, i].join('|')}>
+        return (i === codeLines.length - 1) ? (
+          <Box
+            component="span"
+            key={[line, i].join('|')}
```

Comment:
This can also be written as 

> `${line}|${i}`

Reply:
tried and I got:
`Do not use Array index in keys eslintreact/no-array-index-key`

Reply:
I think that's why I skirted around the lint rule. It's also a pointless rule if we can get around it by using a `join`. @fktt22 maybe we should just remove the lint rule in the future lint config fix.

Reply:
Agreed; the rule should go.